src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/JourneyQueryRepositoryTest.kt

```diff
@@ -106,6 +106,29 @@ internal class JourneyQueryRepositoryTest {
         assertThat(unpricedUniqueJourneys[0].fromNomisAgencyId).isEqualTo("unmappedNomisAgencyId")
     }
 
+
+    @Test
+    fun `2021 prices don't appear in 2020 journeys`() {
+
+        val locationY  = Location(id = UUID.randomUUID(), locationType = LocationType.CO, nomisAgencyId = "locationY", siteName = "apple")
+        locationRepository.save(locationY)
+        priceRepository.save(Price(id = UUID.randomUUID(), fromLocation = locationY, toLocation = gni, priceInPence = 999, supplier = Supplier.SERCO, effectiveYear = 2021))
+
+        val moveWithUpriced2020Journey = standardMove.copy(moveId = "M2")
+        val journeyNotPricedFor2020 = journey(moveId = "M2", journeyId = "J2", billable = true, fromNomisAgencyId = locationY.nomisAgencyId)
+
+        moveRepository.save(moveWithUpriced2020Journey)
+        journeyRepository.save(journeyNotPricedFor2020)
+
+        entityManager.flush()
+
+        val summaries = journeyQueryRepository.journeysSummaryInDateRange(Supplier.SERCO, moveDate, moveDate)
+        assertThat(summaries).isEqualTo(JourneysSummary(2, 999, 0, 1, Supplier.SERCO))
+
+        val unpricedUniqueJourneys = journeyQueryRepository.distinctJourneysAndPriceInDateRange(Supplier.SERCO, moveDate, moveDate)
```

Comment:
Might want to consider renaming the **moveDate**, not immediately obvious it is a 2020 date.